src/main/kotlin/io/sip3/salto/ce/rtcp/RtcpHandler.kt

```diff
@@ -127,6 +141,43 @@ open class RtcpHandler : AbstractVerticle() {
         }
     }
 
+    open fun handleHep(packet: Packet) {
+        val json = JsonObject(packet.payload.toString(Charset.defaultCharset()))
+        if (json.getJsonObject("sender_information").getLong("ntp_timestamp_sec").toInt() != 0
```

Comment:
I think we can start here from: `json.getJsonObject("sender_information")?.let { senderInfo ->` and use it in further calls instead of calling json.getJsonObject("sender_information") multiple times